### PR TITLE
Remove the appliance RPM once the deployment is finished

### PR DESCRIPTION
**Pull request by tiraboschi**

### CODE REVIEW

**Comment by tiraboschi** [comment]
#Fixes https://bugzilla.redhat.com/1676825

File: tasks/create_target_vm/03_hosted_engine_final_tasks.yml [hunk 1]
```diff
@@ -352,6 +352,11 @@
         register: vmremove_result
       - debug: var=vmremove_result
     when: local_vm_f.ansible_facts.ovirt_vms|length > 0
+  - name: Remove ovirt-engine-appliance rpm
+    yum:
+      name: ovirt-engine-appliance
```
Comment (irosenzw) [inline tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: please add also rhvm-appliance for d/s

Comment (tiraboschi) [reply tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: on downstream spec file we already have:
Provides: ovirt-engine-appliance

so yum remove ovirt-engine-appliance will result in

Removing:
 rhvm-appliance

File: tasks/create_target_vm/03_hosted_engine_final_tasks.yml [hunk 2]
```diff
@@ -352,6 +352,11 @@
...
+      state: absent
+    when: he_remove_appliance_rpm is true
```
Comment (kobihk) [inline tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: I think you can remove the "is true"
when: he_remove_appliance_rpm 
will be enough

Comment (tiraboschi) [reply tasks/create_target_vm/03_hosted_engine_final_tasks.yml]: done

**Comment by tiraboschi** [comment]
Verified:
11:58:29 [ INFO  ] TASK [ovirt.hosted_engine_setup : Remove ovirt-engine-appliance rpm]
11:58:39 [ INFO  ] changed: [localhost]

**Pull request merged**